### DISCUSSION TITLE
RFC_READ_TABLE 'FIELDS' values

### DISCUSSION THREAD

**Discussion by Franco-Figueredo** [SAP/node-rfc#255] [General]
Hello!

I would like to know more about the function module RFC_READ_TABLE since I am working with it at the moment. I can pass parameters such as "QUERY_TABLE" to select the table I'll be querying, "DELIMITER" to select the delimiter for the data output and "FIELDS" in order to select the columns with data that's going to be downloaded. For this last parameter, can I pass a collection of items (column names) or I'll be allowed just to past values one by one like:

```
FIELDS: [
   {
      'FIELDNAME': 'COLUMN1'
   },
   {
      'FIELDNAME': 'COLUMN2'
   },
   {
      'FIELDNAME': 'COLUMN2'
   },
]
```

Which is the equivalent of, in VBA:

```
table.AddRow
table(1, 'FIELDNAME') = COLUMN1

table.AddRow
table(2, 'FIELDNAME') = COLUMN2

table.AddRow
table(3, 'FIELDNAME') = COLUMN3
```

Is there a way in JavaScript as well as in VBA to send a collection of items instead of defining a new row and add the field name to the first column? Or should it be done the way its written above?

In VBA I've tried with:

`table.Data = [("COLUMN1", "000000", "000000", "", ""), ("COLUMN2", "000000", "000000", "", "")]`

But had no luck yet. I would like to be capable of reproducing it into JavaScript aswell. Is there any documentation about it?

Thank you!

**Comment by bsrdjan** [comment]
Closed due to inactivity
